pom.xml

```diff
@@ -25,11 +25,7 @@
     <description>aws-sdk</description>
 
     <modules>
-        <module>core</module>
```

Comment:
You should delete the files/folders while you're at it for all of these modules.